FAQ/How to Read an Existing Excel File.md

```diff
@@ -0,0 +1,40 @@
+# How to Read an Existing Excel File
+
+```powershell
+Import-Module ImportExcel
+#Loads the Excel file into a PSCustomObject
+$ExcelFile = Import-Excel "C:\Test\file.xlsx" -WorksheetName "Sheet1" 
+```
+
+## Visual of Data Structure
+The File C:\Test\file.xlsx contains  
+![ExcelFileContents](/images/FAQ_Images/ExcelFileContents.png)
+
+After loading this data into ```$ExcelFile``` the data is stored like:  
+![ExcelFileDebugImg](/images/FAQ_Images/ExcelFileDebugImg.jpg)
+
+## Other Common Operations
+
+### Load a Column
+```powershell
+$SpecificColumn = $ExcelFile."anotherHeader" #loads column with the header "anotherHeader" -- data stored in an array
+```
+
+### Load a Row
+```powershell
+$SpecificRow = $ExcelFile[1] #Loads row at index 1. Index 1 is the first row instead of 0. 
+```
+
+### Map Contents to Hashtable to Interpret Data
+Sometimes mapping to a Hashtable is more convenient to have access to common Hashtable operations. Enumerate a Hashtable with the row's data by:
+```powershell
+$HashTable = @{}
+$SpecificRow= $ExcelFile[2]
+$SpecificRow.psobject.properties | ForEach-Object {$HashTable[$_.Name] = $_.Value}
```

Comment:
Leave the `ForEach-Object {` there and put the rest on a new line, inlcuding the closing `}`. Makes it more readable.

Reply:
Made this change.